Cargo.toml

```diff
@@ -25,5 +25,5 @@ strum = { version = "0.25", features = ["derive"] }
 tokio = { version = "1.29", features = ["full"] }
 tracing = "0.1"
 
-# [patch."https://github.com/stackabletech/operator-rs.git"]
-# stackable-operator = { git = "https://github.com/stackabletech//operator-rs.git", branch = "main" }
+[patch."https://github.com/stackabletech/operator-rs.git"]
```

Comment:
Check that Authclasses are watched!

Reply:
AuthenticationClasses are watched: https://github.com/stackabletech/superset-operator/blob/b5f640f914afadaf7b28168f466e37587fdec063/rust/operator-binary/src/main.rs#L101-L102